content/docs/start/experiments.md

```diff
@@ -19,31 +19,82 @@ once they're no longer needed.
 ## Running experiments
 
 In the previous page, we learned how to tune
-[ML pipelines](/doc/start/data-pipelines) and compare the changes. Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a [new Get Started project][dvcgs] to illustrate
+experimentation features in DVC 2.0
 
 `dvc exp run` makes it easy to change <abbr>hyperparameters</abbr> and run a new
-experiment:
+experiment. We'll use it to find parameters that results in better
+classification performance for [Fashion-MNIST][fmnist] dataset.
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+> You can run these commands in the container we built for this tutorial. It has
+> all the code and data required to run these examples.
+> `docker run -it dvcorg/doc-start:experiments`
```

Comment:
Can we designate pages that have Docker similar to Katacoda (may be make it more visible?) So that we don't repeat the same note over and over again?

Reply:
And to be clear - it can be a note at the end, it would be great to unify though, put docker link into into frontametter and let engine generate it

Reply:
> Can we designate pages that have Docker similar to Katacoda (may be make it more visible?) So that we don't repeat the same note over and over again?

Yes, I prefer that too, and initially I put docker to the front matter, but I later I changed back to include in the content. I mentioned here to keep the PR isolated to this page. Katacoda links are specified in `sidebar.json`. We perhaps need an update for the sidebar code to show the links to docker. 

I'm removing this completely for now.